src/test/java/io/confluent/examples/streams/window/CustomWindowTest.java

```diff
@@ -61,220 +61,243 @@
+                                 // we could support more complex cases with non fixed time
+                                 // window and address daylight savings on daily windows.
+                                 ZonedDateTime.of(2019, 3, 31, 19, 0, 0, 0, zoneWithDST)),
+                      10)
+      );
+      verify(inputValues, expectedValues, zoneWithDST);
     }
 
-    private void verify(final List<MyEvent> inputValues, final List<ExpectedResult> expectedValues, final ZoneId zoneId) {
-
-        final Properties streamsConfiguration = new Properties();
-        streamsConfiguration.put(StreamsConfig.APPLICATION_ID_CONFIG, "sum-lambda-integration-test");
-        streamsConfiguration.put(StreamsConfig.BOOTSTRAP_SERVERS_CONFIG, "dummy:1234");
-        streamsConfiguration.put(StreamsConfig.DEFAULT_KEY_SERDE_CLASS_CONFIG, Serdes.Integer().getClass().getName());
-        streamsConfiguration.put(StreamsConfig.DEFAULT_VALUE_SERDE_CLASS_CONFIG, Serdes.Integer().getClass().getName());
-        // The commit interval for flushing records to state stores and downstream must be lower than
-        // this integration test's timeout (30 secs) to ensure we observe the expected processing results.
-        streamsConfiguration.put(StreamsConfig.COMMIT_INTERVAL_MS_CONFIG, 10 * 1000);
-        streamsConfiguration.put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");
-        // Use a temporary directory for storing state, which will be automatically removed after the test.
-        streamsConfiguration.put(StreamsConfig.STATE_DIR_CONFIG, TestUtils.tempDirectory().getAbsolutePath());
-
-        final Topology topology = buildKafkaStreamTopology(zoneId);
-
-        final TopologyTestDriver testDriver = new TopologyTestDriver(topology, streamsConfiguration);
-
-        injectFakeData(inputValues, testDriver);
-        verifyResults(expectedValues, testDriver);
-
-        testDriver.close();
+    private void verify(final List<TestRecord<String, Integer>> inputValues,
+                        final List<KeyValue<Windowed<Integer>, Integer>> expectedValues,
+                        final ZoneId zoneId) {
+
+      final Properties streamsConfiguration = new Properties();
+      streamsConfiguration.put(StreamsConfig.APPLICATION_ID_CONFIG, "sum-lambda-integration-test");
+      streamsConfiguration.put(StreamsConfig.BOOTSTRAP_SERVERS_CONFIG, "dummy:1234");
+      streamsConfiguration.put(StreamsConfig.DEFAULT_KEY_SERDE_CLASS_CONFIG, Serdes.Integer().getClass().getName());
+      streamsConfiguration.put(StreamsConfig.DEFAULT_VALUE_SERDE_CLASS_CONFIG, Serdes.Integer().getClass().getName());
+      // The commit interval for flushing records to state stores and downstream must be lower than
+      // this integration test's timeout (30 secs) to ensure we observe the expected processing results.
+      streamsConfiguration.put(StreamsConfig.COMMIT_INTERVAL_MS_CONFIG, 10 * 1000);
```

Comment:
This config has no effect now, right?

Reply:
Ack. Seems to be a left over from moving from "EmbeddedKafka" to TDD.